test/unit/api.js

```diff
@@ -0,0 +1,85 @@
+var picc = require('../../js/src/picc');
+var assert = require('assert');
+var extend = require('extend');
+
+var xhr = require('xmlhttprequest');
+// make XMLHttpRequest available to d3
+global.XMLHttpRequest = xhr.XMLHttpRequest;
+
+// use the API_BASE_URL environment variable,
+// or production API by default
+picc.API.url = process.env.API_BASE_URL || 'https://api.data.gov/ed/collegescorecard/v1/';
+
+// XXX you'll need to set the API_KEY environment varible
+// for this to work
+picc.API.key = process.env.API_KEY;
+
+// expected default parameters
+const EXPECTED_DEFAULTS = {
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).